packages/flutter_tools/lib/src/localizations/message_parser.dart

```diff
@@ -0,0 +1,510 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
```

Comment:
It be helpful to include a comment that links to the source of the syntax definition.

Reply:
Fixed!